CHANGELOG.md

```diff
@@ -3,23 +3,48 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [1.7.0](https://github.com/adobe/spectrum-web-components/compare/v1.6.0...v1.7.0) (2025-06-11)
```

Comment:
We'll need to fix this in a different update but just a note that a document should only have 1 h1 heading.